### PR TITLE
Add info on grafana with KES/KMS

### LINKED ISSUE #1057: [RELEASE] Server RELEASE.2023-11-01T01-57-10Z
MinIO Server [RELEASE.2023-11-01T01-57-10Z](https://github.com/minio/minio/releases/tag/RELEASE.2023-11-01T01-57-10Z) has changes that impact the docs:

- [ ] When installing for Docker, config directory now passes as an environment variable from the Docker file rather than requiring a flag ([PR #18317](https://github.com/minio/minio/pull/18317))
- [ ] Grafana now shows KMS info when KES is configured ([PR #18321](https://github.com/minio/minio/pull/18321))

### PR DESCRIPTION
**Pull request by djwfyi**
Closes #1057 .

### CODE REVIEW

File: source/operations/monitoring/grafana.rst [hunk 1]
```diff
@@ -41,6 +41,8 @@ Visualize MinIO metrics with the official MinIO Grafana dashboard for the MinIO
 MinIO provides a Grafana Dashboard for MinIO Server metrics.
 For specifics on the dashboard's configuration, see the `JSON file on GitHub <https://raw.githubusercontent.com/minio/minio/master/docs/metrics/prometheus/grafana/minio-dashboard.json>`__.
 
+If :ref:`KES <minio-sse-data-encryption>` is setup and the MinIO server process has access to the KES metrics, the dashboard includes metrics for the KMS, including status, request error rates, and request success rates.
```
Comment (feorlen) [inline source/operations/monitoring/grafana.rst:44]: Maybe break up the long sentence?
```suggestion
If :ref:`KES <minio-sse-data-encryption>` is configured and the MinIO server process has access to the KES metrics, the dashboard includes metrics for the KMS. Available metrics include status, request error rates, and request success rates.
```

**Review by feorlen — APPROVED** [review]
A reword idea, if you like. I'm happy to approve and you can decide.

File: source/operations/monitoring/grafana.rst (hunk 1 above)
Comment (ravindk89) [reply source/operations/monitoring/grafana.rst:44]: i'll also take a shot here

```suggestion
For MinIO Deployments running with :ref:`Server-Side Encryption <minio-sse-data-encryuption>` (SSE-KMS or SSE-S3), the dashboard includes metrics for the KMS.
These metrics include status, request error rates, and request success rates.
```

I *

**Review by ravindk89 — APPROVED** [review]
LGTM mod nit, of which you can take whomevers or roll your own

File: source/operations/monitoring/grafana.rst (hunk 1 above)
Comment (feorlen) [reply source/operations/monitoring/grafana.rst:44]: Ooo I like Ravind's option. 👍🏻

**Pull request merged**